docs/articles/actors/testing-actor-systems.md

```diff
@@ -179,6 +179,11 @@ The first option is to avoid use of the `context.parent` function and create a c
 
 [!code-csharp[DependentChild](../../../src/core/Akka.Docs.Tests/Testkit/ParentSampleTest.cs?range=44-57)]
 
+###Create the child using the TestProbe
+The `TestProbe` class can directly create child actors using the `ChildActorOf` methods.  
+
+[!code-csharp[TestProbeChild](../../../src/core/Akka.Docs.Tests/Testkit/ParentSampleTest.cs?range=60-64)]
```

Comment:
I'd change this to use `#region` targeting instead, i.e. https://github.com/akkadotnet/akka.net/blob/dev/src/core/Akka.Docs.Tests/Streams/StreamRefsDocTests.cs#L20 - you can actually target the region name using DocFx: https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html#code-snippet-sample

Reply:
Reason why I recommend it: re-organizing the file doesn't break documentation as easily.

Reply:
The latest changes should be using DocFX named references now.  

To help convert the remaining docs, I've written a program to go through all the .md files and automatically convert line references to C# regions.  I think there's a few references that use `#` syntax, so might need to be adapted in the future. 

https://gist.github.com/nagytech/bb07f1c46cda7b88ed8c3a5f4df7e00b